hugo-runtime/src/main/java/hugo/weaving/internal/Hugo.java

```diff
@@ -16,6 +16,9 @@
 
 @Aspect
 public class Hugo {
+
+  private static boolean sEnabled;
```

Comment:
No 's' prefix! Also you'll want this to be `volatile` or threads might not see value changes.